src/core/machine.tsx

```diff
@@ -40,7 +40,7 @@ export default Machine<CoreContext, CoreSchema, CoreEvents>(
           },
           OPEN_REQUEST_DETAILS: {
             actions: ['setSelectedRequest'],
-            target: '.requestDetails',
+            target: '',
```

Comment:
I've been searching about that recently and it might be a good opportunity to ask: which effect an empty target have? Does it tell XState to stay in the same state?

Reply:
yep, exactly! as this is a global event it can happen in any state without forcing a new transition